app/searches/locations_search.rb

```diff
@@ -34,6 +34,53 @@ def search
     search_results
   end
 
+  def exact_match_found?
+    keyword_to_use = keywords || attributes[:keyword]
+
+    if keyword_to_use.blank? && matched_category.blank?
+      return false
+    end
+
+    if matched_category.present?
+      return true
+    end
+
+    return false if keyword_to_use.blank?
```

Comment:
I think this will be covered by the conditional on like 40

Reply:
Yeah it will, I went through a lot of iterations